src/scripts/collapse-expand.js

```diff
@@ -3,10 +3,9 @@
  * @param {Boolean} newState expand or collapse state
  */
 export function modifyAccordionState(e, newState) {
-  const accordionSection = e
-    .composedPath()
-    .find(element => element.classList.contains('qg-accordion'));
-  const articles = accordionSection.querySelectorAll(
+  // composedPath is not SUPPORTED IN IE11, closest is working (supported using Babel polyfill)
```

Comment:
Has come from master branch, as my feature branch was pulled from here? any issues @DanielTurner

Reply:
Procedure is to pull from develop branch and the merge request goes back to develop.